src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/PublicIpAddressTests.ps1

```diff
@@ -278,6 +278,58 @@ function Test-PublicIpAddressCRUD-ReverseFqdn
     }
 }
 
+<#
+.SYNOPSIS
+Tests edit the domain name label of a publicIpAddress with Iptags
+#>
+function Test-PublicIpAddressCRUD-IpTag
+{
+    # Setup
+    $rgname = Get-ResourceGroupName
+    $rname = Get-ResourceName
+    $domainNameLabel = Get-ResourceName
+    $rglocation = Get-ProviderLocation ResourceManagement
+    $resourceTypeParent = "Microsoft.Network/publicIpAddresses"
+    $location = Get-ProviderLocation $resourceTypeParent
+
+    try
+     {
+      # Create the resource group
+      $resourceGroup = New-AzureRmResourceGroup -Name $rgname -Location $rglocation -Tags @{ testtag = "testval" } 
+      
+      $IpTag = New-AzureRmPublicIpTag -IpTagType "FirstPartyUsage" -Tag "/Sql"
+
+      # Create publicIpAddres
+      $actual = New-AzureRmPublicIpAddress -ResourceGroupName $rgname -name $rname -location $location -AllocationMethod Dynamic -DomainNameLabel $domainNameLabel -IpTag $IpTag
+      $publicip = Get-AzureRmPublicIpAddress -ResourceGroupName $rgname -name $rname
+      Assert-AreEqual $publicip.ResourceGroupName $actual.ResourceGroupName
+      Assert-AreEqual $publicip.Name $actual.Name
+      Assert-AreEqual $publicip.Location $actual.Location
+      Assert-AreEqual "Dynamic" $publicip.PublicIpAllocationMethod
+      Assert-AreEqual "Succeeded" $publicip.ProvisioningState
+      Assert-AreEqual $domainNameLabel $publicip.DnsSettings.DomainNameLabel
+
+      # Set publicIpAddress
+      $publicip | Set-AzureRmPublicIpAddress
```

Comment:
What is this doing?

Reply:
piping values to Set-AzureRmPublicIpAddress

Reply:
Got it.